charts/aws-vpc-cni/README.md

```diff
@@ -75,9 +77,11 @@ The following table lists the configurable parameters for this chart and their d
 | `nodeAgent.image.endpoint`   | ECR repository endpoint to use.                    | `ecr`                               |
 | `nodeAgent.image.account`    | ECR repository account number                      | `602401143452`                      |
 | `nodeAgent.image.pullPolicy` | Container pull policy                              | `IfNotPresent`                      |
+| `nodeAgent.image.overrideRepository` | A custom docker image to use (without tag) | `nil`                               |
```

Comment:
image.overrideRepository | Repository override for the image (does not change the tag)